x-pack/plugins/security_solution/public/management/components/artifact_entry_card/components/card_compressed_header.tsx

```diff
@@ -0,0 +1,148 @@
+import { CardExpandButton } from './card_expand_button';
+import { TextValueDisplay } from './text_value_display';
+import { EffectScope } from './effect_scope';
+import { CardActionsFlexItem } from './card_actions_flex_item';
+import { ArtifactInfo } from '../types';
+import { ArtifactEntryCollapsableCardProps } from '../artifact_entry_collapsable_card';
+import { useTestIdGenerator } from '../../hooks/use_test_id_generator';
+import { useCollapsedCssClassNames } from '../hooks/use_collapsed_css_class_names';
+import { usePolicyNavLinks } from '../hooks/use_policy_nav_links';
+import { getEmptyValue } from '../../../../common/components/empty_value';
+
+export interface CardCompressedHeaderProps
+  extends Pick<CommonProps, 'data-test-subj'>,
+    Pick<
+      ArtifactEntryCollapsableCardProps,
+      'onExpandCollapse' | 'expanded' | 'actions' | 'policies'
+    > {
+  artifact: ArtifactInfo;
+}
+
+export const CardCompressedHeader = memo<CardCompressedHeaderProps>(
+  ({
+    artifact,
+    onExpandCollapse,
+    policies,
+    actions,
+    expanded = false,
+    'data-test-subj': dataTestSubj,
+  }) => {
+    const getTestId = useTestIdGenerator(dataTestSubj);
+    const cssClassNames = useCollapsedCssClassNames(expanded);
+    const policyNavLinks = usePolicyNavLinks(artifact, policies);
+
+    const handleExpandCollapseClick = useCallback(() => {
+      onExpandCollapse();
+    }, [onExpandCollapse]);
+
+    return (
+      <EuiFlexGroup responsive={false} alignItems="center" data-test-subj={dataTestSubj}>
+        <EuiFlexItem grow={false}>
+          <CardExpandButton
+            expanded={expanded}
+            onClick={handleExpandCollapseClick}
+            data-test-subj={getTestId('expandCollapse')}
+          />
+        </EuiFlexItem>
+        <EuiFlexItem className={cssClassNames}>
+          <EuiFlexGroup alignItems="center">
+            <EuiFlexItem grow={2} className={cssClassNames} data-test-subj={getTestId('title')}>
+              <TextValueDisplay bold truncate={!expanded}>
+                {artifact.name}
+              </TextValueDisplay>
+            </EuiFlexItem>
+            <EuiFlexItem
+              grow={3}
+              className={cssClassNames}
+              data-test-subj={getTestId('description')}
+            >
+              <TextValueDisplay truncate={!expanded}>
+                {artifact.description || getEmptyValue()}
+              </TextValueDisplay>
+            </EuiFlexItem>
+            <EuiFlexItem grow={1}>
+              <EffectScope policies={policyNavLinks} data-test-subj={getTestId('effectScope')} />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiFlexItem>
+        <CardActionsFlexItem actions={actions} data-test-subj={getTestId('actions')} />
+      </EuiFlexGroup>
+    );
+  }
+);
+CardCompressedHeader.displayName = 'CardCompressedHeader';
+
+const ButtonIconPlaceHolder = styled.div`
+  display: inline-block;
+  width: 24px;
+  height: 24px;
```

Comment:
I guess I can use Eui's Icon them value. I used hard coded values because there are no them props for the width/height of a `EuiIconButton` and the measurement should be as accurate as possible (in order to not distort the grid's list header).  Either way, a change to the theme would break the alignment, so I will change it to what you suggest 😄